content/en/about.html

```diff
@@ -76,9 +76,12 @@ <h2>Policies</h2>
   <div class="container">
     <h2>Open Source Repositories</h2>
     <p>Most of the code we use to operate Divvi Up is open source.</p>
+    <p>Divvi Up is based on open standards and most of the code we use to operate it is open source.</p>
     <ul>
-      <li><a href="https://github.com/divviup/prio-server">Prio Server on Github</a></li>
-      <li><a href="https://github.com/divviup/libprio-rs">Prio Rust Implementation on Github</a></li>
+      <li>Divvi Up implements the Distributed Aggregation Protocol (DAP) (<a href="https://github.com/ietf-wg-ppm/ppm-specification">specification on Github</a>)</li>
+      <li>DAP is built on Verifiable Distributed Aggregation Functions (VDAF), (<a href="https://github.com/cfrg/draft-irtf-cfrg-vdaf">specification on Github</a>)</li>
+      <li><a href="https://github.com/divviup/libprio-rs">VDAF Rust Implementation on Github</a></li>
+      <li><a href="https://github.com/divviup/janus">Janus is the DAP implementation that powers Divvi Up</a></li>
       <li><a href="https://github.com/divviup/janus">Janus Server on Github</a></li>
       <li><a href="https://github.com/abetterinternet/ppm-specification">DAP Specification on Github</a></li>
```

Comment:
Drop this line too -- the link on this line ultimately resolves to https://github.com/ietf-wg-ppm/ppm-specification which is also linked to by one of the new lines introduced in this PR.